app/models/miq_provision_virt_workflow.rb

```diff
@@ -620,12 +612,6 @@ def get_source_and_targets(refresh = false)
     result
   end
 
-  def source_valid?
```

Comment:
This was added by @lfu in PR #2947.  As I recall the UI was going to use this to validate the requests attached to a service.  @lfu do you know if that was every fully implemented?